src/pages/[productSlug]/tutorials/[...tutorialSlug]/index.tsx

```diff
@@ -88,7 +88,7 @@ async function _getStaticProps({
 	if (!props) {
 		return { notFound: true }
 	}
-	return props
+	return { props, revalidate: __config.dev_dot.revalidate }
```

Comment:
Just an opinion but I do prefer the meaningful `getStaticProps` logic on the page file itself. In other places we return revalidate with the same function that we create the props on. Personally, I find it helpful to go directly to the page file to check out paths / props config like this are being set.